web/src/beta/lib/core/StoryPanel/SelectableArea/index.tsx

```diff
@@ -55,49 +53,44 @@ const SelectableArea: React.FC<Props> = ({
   setEditMode,
   onEditModeToggle,
   onSettingsToggle,
-  onClick,
-  onClickAway,
   onRemove,
   onPropertyUpdate,
   onPropertyItemAdd,
   onPropertyItemMove,
   onPropertyItemDelete,
 }) => {
-  const { showPadding, setShowPadding, handleClickAway } = useHooks({
+  const { showPadding, setShowPadding } = useHooks({
     editMode,
     isSelected,
     setEditMode,
-    onClickAway,
   });
 
   return !isEditable ? (
     <>{children}</>
   ) : (
-    <ClickAwayListener enabled={isSelected} onClickAway={handleClickAway}>
-      <Wrapper isSelected={isSelected} noBorder={noBorder} onClick={onClick}>
-        <ActionPanel
-          title={title}
-          icon={icon}
-          isSelected={isSelected}
-          showSettings={showSettings}
-          showPadding={showPadding}
-          editMode={editMode}
-          propertyId={propertyId}
-          panelSettings={panelSettings}
-          dndEnabled={dndEnabled}
-          position={position}
-          setShowPadding={setShowPadding}
-          onEditModeToggle={onEditModeToggle}
-          onSettingsToggle={onSettingsToggle}
-          onRemove={onRemove}
-          onPropertyUpdate={onPropertyUpdate}
-          onPropertyItemAdd={onPropertyItemAdd}
-          onPropertyItemMove={onPropertyItemMove}
-          onPropertyItemDelete={onPropertyItemDelete}
-        />
-        {children}
-      </Wrapper>
-    </ClickAwayListener>
+    <Wrapper isSelected={isSelected} noBorder={noBorder}>
```

Comment:
I don't think you should have removed the onClick on the Wrapper. Now the only way to select the Page is on the empty space below the page, not the page itself.